vm/src/anystr.rs

```diff
@@ -376,7 +376,7 @@ pub trait AnyStr {
         }
     }
 
-    fn py_splitlines<FW, W>(&self, options: SplitLinesArgs, into_wrapper: FW) -> Vec<W>
+    fn py_bytes_splitlines<FW, W>(&self, options: SplitLinesArgs, into_wrapper: FW) -> Vec<W>
```

Comment:
Did you mean to make this function so that it's no longer generic over both `str`s and bytes? If so, then it probably doesn't belong in `anystring.rs` anymore.

Reply:
I guess, it doesn't have any uses in the `str` context any more.